Samples/WinMLSamplesGallery/WinMLSamplesGallery/Pages/SampleBasePage.xaml

```diff
@@ -0,0 +1,67 @@
+<Page
+    x:Class="WinMLSamplesGallery.SampleBasePage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:WinMLSamplesGallery"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Page.Resources>
+        <DataTemplate x:Key="DocsListViewTemplate" x:DataType="local:SampleDoc">
+            <TextBlock Margin="10,20,0,0" FontSize="17">
+                <Hyperlink TextDecorations="None" NavigateUri="{Binding link}">
+                    BatchSizeOverride API
+                </Hyperlink>
+            </TextBlock>
+        </DataTemplate>
+    </Page.Resources>
+
+    <ScrollViewer>
+        <Grid>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="0.7*"></ColumnDefinition>
+                <ColumnDefinition Width="0.3*"></ColumnDefinition>
+            </Grid.ColumnDefinitions>
+            <StackPanel Grid.Row="0" Grid.Column="0" Padding="40,40">
+                <TextBlock FontWeight="Bold" FontSize="25" Text="{x:Bind sample.Title}" />
+                <TextBlock FontSize="16" Margin="0,15,0,0" Text="{x:Bind sample.Description}" />
+                <Frame x:Name="SampleFrame" />
+            </StackPanel>
+
+            <StackPanel Grid.Row="0" Grid.Column="1" Padding="0,40,40,0" HorizontalAlignment="Right">
+                <TextBlock FontWeight="Bold" FontSize="20">View page code on GitHub</TextBlock>
+                <TextBlock Margin="10,20,0,0" FontSize="16">
+                    <Hyperlink TextDecorations="None" NavigateUri="{x:Bind sample.XAMLGithubLink}">
+                        XAML Source Code
+                    </Hyperlink>
+                </TextBlock>
+                <TextBlock Margin="10,20,0,0" FontSize="16">
+                    <Hyperlink TextDecorations="None" NavigateUri="{x:Bind sample.CSharpGithubLink}">
+                        C# Source Code
```

Comment:
what is the difference between XAML source code and C# source code? why isn't it just "source code"?

Reply:
Every sample page has a xaml file (frontend) and c# file (e.g. Batching.xaml and Batching.xaml.cs) so these are separate links to each of them